mta/carbonio-mta

```diff
@@ -65,6 +65,9 @@ fi
 chown ${SERVICE_USER}:${SERVICE_GROUP} ${SERVICE_BASE_DIR}/token
 chmod 0600 ${SERVICE_BASE_DIR}/token
 
+# register prepared query
```

Comment:
can you call /opt/zextras/common/bin/curl? (we use an up-to-date version). We already provide a curl executable and its installation is guaranteed by carbonio-mariadb for mta nodes.

Reply:
done